### ISSUE TITLE
ENH: Incorporate search in the theme

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/quantecon-book-theme#158]
The base `sphinx-book-theme` has search enabled. 

It  would be nice to have search enabled for our sites. 

@DrDrij we will need to think about where to put it. I was wondering if a `magnifying glass` on the top right set of icons that could expand into a search bar perhaps?

**Comment by AakashGfude** [comment]
@jstac @shlff The issue, which Shu was talking about today.

**Comment by mmcky** [comment]
@jstac talked with @AakashGfude today about adding this.

As a first iteration, @AakashGfude will add a search magnifying glass jus to the right of the "QuantEcon logo in the title bar that when clicked on will expand into a search text entry box.

Once that mockup is ready we can review style with @DrDrij and/or yourself.

**Comment by jstac** [comment]
Thanks @mmcky , sounds like a good plan.

**Comment by mmcky** [comment]
@AakashGfude is this hard to implement? I know it's available in sphinx-book-theme. Would you mind checking sometime this afternoon if this is hard to do as it would be a nice to have if we can activate it.

**Comment by AakashGfude** [comment]
I will have a look @mmcky .

**Comment by mmcky** [comment]
This can be closed with `0.5.0` release - some minor updates coming in `0.5.1`
